src/App.tsx

```diff
@@ -1,11 +1,25 @@
-import React from 'react';
-import Mainform from './components/MainForm/Mainform';
-import './App.css';
+import React from "react";
+import Mainform from "components/MainForm/Mainform";
+import Splash from "components/LoadingScreen/Splash";
+import { BrowserRouter, Route } from "react-router-dom";
+import "./App.css";
+import { Provider } from "react-redux";
+import store from "store/reducers/mainForm";
+
 
 function App() {
   return (
     <div className="App">
-     <Mainform/>
+      <Provider store={store}>
+      <BrowserRouter>
+        <Route exact path="/">
+          <Mainform />
+        </Route>
+        <Route path="/Loading">
+          <Splash />
```

Comment:
You should name this comment something 'meaningful', splash doesn't makes sense

Reply:
this component name changed with "Loader"